test/NuGet.Core.Tests/NuGet.Protocol.Tests/Core/Types/PackageSearchMetadataBuilderTests.cs

```diff
@@ -24,15 +24,15 @@ public void LocalPackageInfo_NotNull()
             Assert.True(copy1 is PackageSearchMetadataBuilder.ClonedPackageSearchMetadata);
 
             var clone1 = (PackageSearchMetadataBuilder.ClonedPackageSearchMetadata)copy1;
-            Assert.NotNull(clone1.PackageReader);
+            Assert.NotNull(clone1.PackagePath);
```

Comment:
I suppose this test was originally focused on the PackageReader not copying correctly. I don't think it's very interesting for a String path. 
I'd either remove this test and make sure some other test validates all cloned properties actually got cloned, or changes this to something like `clone1.PackagePath.Equals(clone2.PackagePath)` ?

Reply:
fixed in next iteration.